### PR TITLE
[Pages] Minimum NODE_VERSION for Next.js sites

### LINKED ISSUE #3818: Insufficient information provided in deploying a Next.js site
### Existing documentation URL(s)

- https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/#deploying-with-cloudflare-pages

### Section that requires update

> Deploying with Cloudflare Pages

### What needs to change?

Next.js has a dependency that requires node version 12.20.0 or higher. The version of node provided by the default Pages configuration is 12.18.0.

### How should it change?

There should be a paragraph added to inform users about this and how-to override the default node version setting by pointing the user to the documentation that covers this (https://developers.cloudflare.com/pages/platform/build-configuration/#language-support-and-tools).

### Additional information

existing-docs

### PR DESCRIPTION
**Pull request by KianNH**
Closes https://github.com/cloudflare/cloudflare-docs/issues/3818

Reference: https://nextjs.org/docs/upgrading

### CODE REVIEW

**Comment by KianNH** [comment]
@Ekwuno looks like the issue was assigned to you at first - you able to give this a look over? Cheers!

File: content/pages/framework-guides/deploy-a-nextjs-site.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ Deploy your site to Pages by logging into the [Cloudflare dashboard](https://das
 
 </div>
 
+{{<Aside type="warning" header="Important">}}
+
+Next.js requires Node.js v12.22.0 or later to build successfully. You must expand the **Environment Variables (advanced)** section and add a `NODE_VERSION` variable with a value of `12.22.0` or greater.
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-nextjs-site.md]:
```suggestion
Next.js requires Node.js v12.22.0 or later to build successfully. To set your Node version, go to **Settings** in your Workers project > **Environment Variables (advanced)** section and add a `NODE_VERSION` variable with a value of `12.22.0` or greater.
```

Comment (KianNH) [reply content/pages/framework-guides/deploy-a-nextjs-site.md]: Should I make a PR to do the same on Astro?

https://developers.cloudflare.com/pages/framework-guides/astro/#deploying-with-cloudflare-pages

Comment (deadlypants1973) [reply content/pages/framework-guides/deploy-a-nextjs-site.md]: @KianNH If it has the same requirement, then I would say yes.

Comment (KianNH) [reply content/pages/framework-guides/deploy-a-nextjs-site.md]: It has the same aside, just with the original `You must expand the` wording - https://github.com/cloudflare/cloudflare-docs/blob/7cba71ec9f639e256acc0ce7e064befead2d6bfa/content/pages/framework-guides/astro.md?plain=1#L72

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/framework-guides/deploy-a-nextjs-site.md (hunk 1 above)
Comment (deadlypants1973) [reply content/pages/framework-guides/deploy-a-nextjs-site.md]: ah yes I remember this. Let's leave it as is then.

**Pull request merged**